scripts/actions/toggleStats.js

```diff
@@ -0,0 +1,7 @@
+import * as types from '../constants/ActionTypes';
+
+export function toggleStats() {
+  return {
+      type: types.TOGGLE_STATS
```

Comment:
> Is there an easier way (e.g.: less boilerplate) for UI actions that modify state but don't have state themselves?

Not sure exactly what you mean by this, but I think I know what you're trying ask.

Some purists would argue that components should ever have state, in other words, *all* data related to state should live in our store. All the data a component would need for its render method would come from `props`.

I personally think that approach is too rigid. There are certain cases where it's perfectly fine to use `state` and not move our data to the global store (which requires a an `action`, a `constant` (action type), and a `reducer`.

For example, if we have dropdown menu and we need to save an `isOpen` flag, it would make little sense to keep this flag in our `store` since the only component that would need access to this data, would be the component itself.

In this case of `toggleStats`, you've done it right. A pretty reasonable test to whether we keep some piece of data in our store or in the component, is to ask whether there is more than one component that requires this data. In this case, the answer is yes, since both the `Nav` and `SongsCards` components both need to know about `toggleStats`. It makes perfect sense to have this live in our global store and so it needs a corresponding `action`, `constant`, and `reducer` since that is the only way to mutate our global store. 

Now, I can see how this appears to be overkill, since you end up creating `actions/toggleStats.js` and `reducers/toggleStats.js`. But let's say instead we had a few different things that the user can toggle besides `stats` (like `desktop notifications`) and we can group them all in `actions/settings` and `reducers/settings`. In my opinion, it's not too much boilerplate.